llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -0,0 +1,651 @@
+//===- llvm-cas-object-format.cpp - Tool for the CAS object format --------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/ADT/BitVector.h"
+#include "llvm/ADT/ScopeExit.h"
+#include "llvm/CAS/HierarchicalTreeBuilder.h"
+#include "llvm/CAS/TreeSchema.h"
+#include "llvm/CASUtil/Utils.h"
+#include "llvm/MCCAS/MCCASObjectV1.h"
+#include "llvm/RemoteCachingService/RemoteCachingService.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Error.h"
+#include "llvm/Support/FileOutputBuffer.h"
+#include "llvm/Support/FileSystem.h"
+#include "llvm/Support/FormatVariadic.h"
+#include "llvm/Support/MemoryBuffer.h"
+#include "llvm/Support/Path.h"
+#include "llvm/Support/Program.h"
+#include "llvm/Support/StringSaver.h"
+#include "llvm/Support/ThreadPool.h"
+#include "llvm/Support/raw_ostream.h"
+#include <memory>
+
+using namespace llvm;
+using namespace llvm::cas;
+using namespace llvm::casobjectformats;
+
+#ifndef NDEBUG
+    // With assertions enabled do a check we get deterministic CASID for
+    // ingestion.
+    constexpr unsigned DefaultRepeats = 1;
```

Comment:
We don't need this. This is for JITObject conversion (which happens inside this tool) and check the conversion is deterministic and correct.
There are no more built-in conversion in this tool anymore, so we don't need this.

Reply:
Not addressed.